src/pages/ExtensionDashboardPage.tsx

```diff
@@ -7,22 +7,32 @@ import { CREATE_PROJECT_ROUTE, SET_GOAL_ROUTE } from "../App";
 import { useEffect, useState } from "react";
 import { UserGoal } from "../models/userModels";
 import { getPersistedUserGoal } from "../storage/userGoalStorage";
+import { getPersistedUserWordCount } from "../storage/entryStorage";
 
 export default function ExtensionDashboardPage() {
   const [userGoal, setUserGoal] = useState<UserGoal | null>({
     goal_count: 0,
     goal_period: "daily",
-    period_start: new Date(),
+    period_start: Date.now(),
   });
+  const [wordCount, setWordCount] = useState<number | null>(0);
   const [projectTitle, setProjectTitle] = useState(PROJECT_TITLE_NONE);
 
   useEffect(() => {
-    loadUserGoal();
-  }, []);
+    loadDashboard();
+    chrome.storage.onChanged.addListener(() => {
```

Comment:
Should this be in a separate effect? We probably don't want it to add another listener every time word count changes

Reply:
good catch, this `useEffect` only needs to fire once so I'll get rid of the `wordCount` in the array